honeycomb-core/src/twomap.rs

```diff
@@ -1289,14 +1288,15 @@ impl<const N_MARKS: usize, T: CoordsFloat> CMap2<N_MARKS, T> {
     ///
     pub fn build_all_faces(&mut self) -> usize {
         self.faces.clear();
+        let mut marked = BTreeSet::<DartIdentifier>::new();
```

Comment:
So this is the place where marks where used?

Reply:
Yes, in the reference implementation / algorithm, marks also were used for orbits but it doesn't really makes sense for a practical Rust implementation IMO.